### DISCUSSION TITLE
404 Deploying Express App

### DISCUSSION THREAD

**Discussion by tydolla00** [vercel/community#3707] [Help] [answered]
### Summary

Hopefully, the documentation on deploying a serverless express app can change. I'm trying to deploy an express app and every route I hit returns a 404 not found. I've tried every method, many vercel.json configurations, and nothing has worked. Any help would be great. My repo https://github.com/tydolla00/Topix . 
Vercel.json
<img width="415" alt="image" src="https://github.com/vercel/community/assets/90355178/b93a590a-9f02-4b60-bc8d-cfd6b1bc16ac">
File structure 
<img width="217" alt="image" src="https://github.com/vercel/community/assets/90355178/88e9296d-f0b3-47fb-a06b-adb5da22dff7">
Package.json
<img width="537" alt="image" src="https://github.com/vercel/community/assets/90355178/a6588da8-97c9-47fc-aa9c-5ffb0ffddcb0">

### Steps to Reproduce

Clone repo, deploy server to vercel.

**Comment by amyegan** [comment, marked as answer]
Hey @tydolla00. Deploying to a serverless environment is a bit different from a traditional server. You can learn more about setting up an Express site for use with Serverless Functions in this guide: https://vercel.com/guides/using-express-with-vercel

It looks like you have a monorepo, so this page may also be useful: https://vercel.com/guides/using-express-with-vercel
